WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/IconListItem.swift

```diff
@@ -10,19 +10,42 @@ struct IconListItem: View {
         /// Base64 icon
         case base64(UIImage)
 
+        case adaptiveBase64(universal: UIImage, dark: UIImage?)
```

Comment:
I am wondering, is calling these parameters universal for light specific enough? After all, we could have some light mode icons that are not universal (they wouldn't look good on dark mode).

Reply:
Actually, probably both places are good.  We may start making these icons from other places, they're already used in Shipping as well as here.

I could make it `adaptiveBase64(anyAppearance: UIImage, dark: UIImage?)` if you think that's better than `universal`?

Reply:
> We could... but they would not be used in an inappropriate context if a dark mode image was provided. I did it this way to raise awareness that if there's no dark image, the light one would be used in that context.

That's true, we have the same in the xcassets file, Appearances are None, Any-Dark, or Any-Light-Dark. Using `adaptiveBase64(anyAppearance: UIImage, dark: UIImage?) ` as you suggested here would be an improvement indeed 👍